core/tests/integration/inputs/pattern-matching/default_value_array_merge_conflict.ncl

```diff
@@ -0,0 +1,7 @@
+# test.type = 'error'
+#
+# [test.metadata]
+# error = 'EvalError::BlameError'
```

Comment:
Why does this fail with a blame error? Shouldn't it be a merge conflict because of the two default values?

Reply:
Theoretically you're right, but merge for arrays is implemented as `array1 & array2 ~ std.contract.Equals array1 some_label array2`. So, technically, when that fails, the error is represented internally as a blame error (and probably reported as such, but at least with custom notes? I don't remember)

Reply:
Ah, of course. It is a merge failure but we're checking equality lazily via a contract that fails.